.github/workflows/documentation.yml

```diff
@@ -55,13 +55,22 @@ jobs:
         env:
           EARTHDATA_USERNAME: ${{ secrets.EDL_USERNAME }}
           EARTHDATA_PASSWORD: ${{ secrets.EDL_PASSWORD }}
-
-      - name: Deploy
-        if: |
-          github.event_name == 'push'
-          && (github.ref == 'refs/heads/main' || github.ref == 'ref/heads/documentation')
-          && github.repository == 'nsidc/earthaccess'
-        uses: peaceiris/actions-gh-pages@v3
+      - name: Upload site artifact
+        uses: actions/upload-pages-artifact@v1
```

Comment:
Looks like there's a `v2` release for this action (and the `deploy-pages@` one too). Any reason to use `v1` over `v2`?

Reply:
Nope, was just lack of awareness and copy/pasting :)